tests/integration/test_permissions.py

```diff
@@ -58,6 +58,7 @@ def _create_and_index_record(record):
 def sample_record(app):
     record = {
         '$schema': 'http://localhost:5000/schemas/records/hep.json',
+        'authors': [],
```

Comment:
Why this empty field? All throughout the code-base we are stripping away empty values (i.e. empty lists, empty objects, empty strings).